apps/client/src/app/components/user-account-access/create-or-update-access-dialog/create-or-update-access-dialog.html

```diff
@@ -59,6 +59,80 @@ <h1 mat-dialog-title>
         </mat-form-field>
       </div>
     }
+
+    @if (showFilterPanel) {
+      <div class="mt-4">
+        <h2 class="mb-3" i18n style="font-size: 1.125rem; font-weight: 500">
+          Filter Settings (Optional)
+        </h2>
+        <p class="text-muted mb-3" i18n style="font-size: 0.875rem">
+          Configure which accounts, holdings, tags, and asset classes will be
+          visible in this public access. Leave empty to show all data.
+        </p>
+
+        <div class="mb-3">
```

Comment:
Why don't you reuse the component from #5605 you have created exactly for this use case? 😱